packages/ledger/src/ledger.canister.ts

```diff
@@ -1,35 +1,34 @@
-import { createServices, toNullable } from "@dfinity/utils";
+import type { QueryParams } from "@dfinity/utils";
```

Comment:
Maybe rename the file to `icrc1-ledger.canister.ts`?

Reply:
Why not, will also drop the `1`

Reply:
On the other side, the package / lib is `@dfinity/ledger` so it follows it and I rather like to publish `@dfinity/ledger` than ``@dfinity/icrc-ledger`. So I would keep `ledger.canister.ts`. Agree?

Reply:
Yes, that's ok.